src/ecalc/libraries/libecalc/common/libecalc/dto/result/results.py

```diff
@@ -134,10 +139,17 @@ class PumpModelResult(ConsumerModelResultBase):
     operational_head: Optional[List[float]]
 
 
-class CompressorModelResult(ConsumerModelResultBase, CompressorTrainResult):
+class CompressorModelResult(ConsumerModelResultBase):
     componentType: Literal[ComponentType.COMPRESSOR]
+    # max_standard_rate: Optional[Union[List[Optional[float]], List[List[Optional[float]]]]]
+    failure_status: List[Optional[CompressorTrainCommonShaftFailureStatus]]
     requested_inlet_pressure: TimeSeriesFloat
     requested_outlet_pressure: TimeSeriesFloat
+    rate_sm3_day: TimeSeriesRate
```

Comment:
I guess we need to name the attribute `rate_sm3_day` wrt API or legacy name? But with e.g. TimeSeriesRate, then information about unit is in that object, and we should therefore not need to provide units etc in the name any more...

```suggestion
    rate: TimeSeriesRate
```

? rate or `gas_rate` ? However, if possible, we should probably change this all over the place in one go, instead of here and there?

Reply:
Have changed to rate in the dto, however, the dto get values from core (CompressorTrainResult) where it is still rate_sm3_day.